proto/stat.proto

```diff
@@ -105,7 +106,8 @@ message ObsTimeSeries {
 // and place type, for given measured property, observation period, stat type,
 // observation date, pop type, and an optional list of pop PVs.
 message ObsCollection {
-  repeated SourceSeries source_cohorts = 2;
+  repeated SourceSeries source_cohorts = 5;
+  reserved 1, 2, 3, 4;
```

Comment:
For this message, no change from the old code except the number of the field?

Reply:
So no need to change this message?

Reply:
This is to match the g3 proto too